spyder/widgets/fileswitcher.py

```diff
@@ -770,6 +770,11 @@ def setup(self):
         # Set position according to size
         self.set_dialog_position()
 
+    def show(self):
+        """Override Qt method."""
+        self.setup()
```

Comment:
Please @jnsebgosselin could you add here a reference to the issue, just for tracking :) . Also thanks a lot for all the work in the `Fileswitcher`! :+1:

Reply:
@jnsebgosselin, please address @dalthviz request, so we can merge this one.